warehouse/oidc/views.py

```diff
@@ -284,4 +284,17 @@ def mint_token(
                 "publisher_url": publisher.publisher_url(),
             },
         )
+
+    if isinstance(publisher, GitHubPublisher) and claims:
```

Comment:
Might be worth adding a `NOTE` or similar comment here explaining that this is for temporary metrics collection 🙂

Reply:
Good point, fixed!

Reply:
Perfect!